language/stdlib/modules/Libra.move

```diff
@@ -551,7 +564,8 @@ module Libra {
             mint_events: Event::new_event_handle<MintEvent>(account),
             burn_events: Event::new_event_handle<BurnEvent>(account),
             preburn_events: Event::new_event_handle<PreburnEvent>(account),
-            cancel_burn_events: Event::new_event_handle<CancelBurnEvent>(account)
+            cancel_burn_events: Event::new_event_handle<CancelBurnEvent>(account),
+            exchange_rate_update_events: Event::new_event_handle<ToLBRExchangeRateUpdateEvent>(account)
```

Comment:
Hopefully our tests will catch this, but just wanted to double-check that this isn't changing any important event keys for handles created later in genesis...

Reply:
Our tests do indeed catch this :) (see the change to `block_metadata.rs`).